### PR TITLE
[[ Benchmarks ]] Add benchmarks makefile

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: benchmarks/Makefile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+################################################################
+# Engine benchmarks
+################################################################
+
+lcs-benchmark: $(LCS_ENGINE)
+	@rm -f $(LCS_LOG)
+	@cmd="$(LCS_CMD)"; \
+	echo "$$cmd" $(_PRINT_RULE); \
+	$$cmd
```
Comment (peter-b) [inline benchmarks/Makefile]: @livecodeali GNU make can behave weirdly if the makefile doesn't end with a linefeed.

Comment (livecodeali) [inline benchmarks/Makefile]: Thanks

**Comment by peter-b** [comment]
@livecode-vulcan review ok 2fda692

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 2fda692b93e3f78b7b0ced424990a01261b98930

**Comment by livecode-vulcan** [comment]
😎 test success 2fda692b93e3f78b7b0ced424990a01261b98930
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2775)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2784)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2770)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2786)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2792)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2796)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2814)

**Pull request merged**